tests/integration/test_setuptools.py

```diff
@@ -14,10 +14,16 @@
 """Integration tests related to building the package."""
 import re
 import subprocess
+import sys
 from pathlib import Path
 from zipfile import ZipFile
 
+import pytest
 
+
+@pytest.mark.skipif(
+    sys.platform == "win32", reason="build module not installed on Windows CI"
```

Comment:
curious; is the `build` dep in pyproject.toml just skipped then?
```
[project.optional-dependencies]
dev = [
    "build",

Reply:
When I remove the skipif it breaks on Windows CI: https://github.com/canonical/starbase/actions/runs/5392038998/jobs/9789800442?pr=107

Looking further though, I'm betting `python3` isn't the venv python on Windows for... reasons. So I'm trying with `sys.executable`, which should fix that. If it doesn't break other things that's probably the best way forward.